common-module/src/main/java/domain/model/User.java

```diff
@@ -0,0 +1,71 @@
+package domain.model;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import org.hibernate.annotations.CreationTimestamp;
+import domain.status.UserStatus;
+
+@Entity
+public class User {
+
+  @Id
+  @GeneratedValue
+  private long id;
+
+  private String userName;
+  private String email;
+  private String accessToken;
+
+  @Column(nullable = false, updatable = false)
+  @CreationTimestamp
+  private LocalDateTime createdAt;
+  private LocalDateTime updatedAt;
+
+  private String nickName;
+  private int likes = 0;
+  private int dislikes = 0;
+  private int reportPoints = 0;
+
+  @Enumerated(value = EnumType.ORDINAL)
+  private UserStatus status = UserStatus.ACTIVE;
+
+  @OneToOne(fetch = FetchType.LAZY, mappedBy = "user")
+  private Address address;
+
+  @OneToOne(fetch = FetchType.LAZY, mappedBy = "user")
+  private Category categoty;
+
+  @OneToOne(fetch = FetchType.LAZY, mappedBy = "user")
+  private Tag tag;
```

Comment:
여러 사용자가 하나의 태그에 매핑할 수 있으므로 ```ManyToOne``` 이 맞을 것 같네요.
```suggestion
  @ManyToOne(fetch = FetchType.LAZY)
  private Tag tag;
```

Reply:
다시 보니까 즐겜러는 유저 소개에 쓰는 것이 아니라 게시물에만 쓰네요. 이 부분은 지우도록 하겠습니다